notebooks/community/model_garden/model_garden_pytorch_pic2word.ipynb

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
<p>Add cleanup section to end of notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Done